adapter/src/main/java/org/jvalue/ods/adapterservice/datasource/api/amqp/AmqpHandler.java

```diff
@@ -0,0 +1,63 @@
+package org.jvalue.ods.adapterservice.datasource.api.amqp;
+
+import org.jvalue.ods.adapterservice.config.RabbitConfiguration;
+import org.jvalue.ods.adapterservice.datasource.event.DatasourceConfigEvent;
+import org.jvalue.ods.adapterservice.datasource.event.DatasourceImportedEvent;
+import org.jvalue.ods.adapterservice.datasource.event.ImportFailedEvent;
+import org.jvalue.ods.adapterservice.datasource.model.Datasource;
+import org.springframework.amqp.AmqpException;
+import org.springframework.amqp.rabbit.core.RabbitTemplate;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.io.Serializable;
+
+@Service
+public class AmqpHandler {
```

Comment:
Rename to AmqpPublisher?
Divide into two classes, for config and import? (not necessary)